modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,34 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
```

Comment:
This is out of scope for this update (since the file was already like this), but just an observation that this is a really large module with a bunch of subsections that should probably be their own modules. Some of the subsections even have the ".Procedure" heading as per the procedure module template. 

At some point, someone should take a look at this and try to break it up properly into modules, and have each of them formatted so that they are clearly for the type (procedure, concept, reference).